.github/workflows/ci.yml

```diff
@@ -212,23 +206,51 @@ jobs:
         name: packages-v${{ steps.versions.outputs.product_full_version }}
         if-no-files-found: error
         path: "GeneratedNuGetPackages/${{ steps.versions.outputs.product_configuration }}/*.*nupkg"
+    - name: Upload SystemTests
+      uses: actions/upload-artifact@v4
+      with:
+        name: SystemTests
+        if-no-files-found: error
+        path: Tests/Reqnroll.SystemTests/bin/
+    - name: Upload Specs
+      uses: actions/upload-artifact@v4
+      with:
+        name: Specs
+        if-no-files-found: error
+        path: Tests/Reqnroll.Specs/bin/
 
   specs:
     runs-on: ubuntu-latest
     needs: build
+    env:
+      # Disable telemetry, because we use many small projects
+      DOTNET_CLI_TELEMETRY_OPTOUT: 1
+      REQNROLL_TELEMETRY_ENABLED: 0
 
     steps:
     - uses: actions/checkout@v4
-    - name: Restore dependencies
-      run: dotnet restore
-    - name: Install Test Report Dependencies
-      run: | 
-        dotnet add ./Tests/Reqnroll.Specs/Reqnroll.Specs.csproj package GitHubActionsTestLogger
-    - name: Build
-      run: dotnet build --no-restore ${{ needs.build.outputs.build_params }}
+    - name: Set Runner ImageVersion to Env
+      shell: pwsh
+      run: |
+        Write-Output "ImageVersion=$env:ImageVersion" >> $env:GITHUB_ENV
+    - name: Cache RRC folder
+      uses: actions/cache@v4
+      with:
+        path: /tmp/RRC
+        key: RRC-Specs-${{ runner.os }}-${{ env.ImageVersion }} # Use the runner version as key, so if new dependencies get installed we get a new cache entry
```

Comment:
What actually gets put in this cached folder?

Reply:
We need to create solution and project files when running the tests. Since #294, these files have been cached in the RRC folder. The cache contains these files.